src/Routing/Filter/DebugBarFilter.php

```diff
@@ -213,6 +207,40 @@ public function afterDispatch(Event $event)
     }
 
     /**
+     * Serialization wrapper for toolbar data
+     *
+     * @param array $data Panel data to serialize.
+     * @return string serialized panel data
+     */
+    protected function _serialize($data)
+    {
+        $response = [];
+        foreach ($data as $key => $value) {
+            if (is_array($value)) {
+                $response[$key] = [];
+                foreach ($value as $k => $v) {
+                    try {
+                        serialize($v);
+                        $response[$key][$k] = $v;
+                    } catch (\Exception $e) {
+                        $response[$key][$k] = $e->getMessage();
+                        $response['error'] = $e->getMessage();
+                    }
+                }
+            } else {
+                try {
+                    serialize($value);
+                    $response[$key] = $value;
+                } catch (\Exception $e) {
+                    $response['error'] = $e->getMessage();
+                }
+            }
+        }
+
+        return serialize($response);
```

Comment:
Given that only the variables panel is the problem, couldn't this go there i stead of being applied to all panels?